include/file.iol

```diff
@@ -68,23 +68,30 @@ type ListRequest:void {
 	.order?: void {
 		.byname?: bool
 	}
+	.fileInfo?: bool // it returns also file infos. Default is false
 }
 
 type ListResponse:void {
-	.result[0,*]:string
+	.result[0,*]:string {
```

Comment:
What you mean is probably:

```
.result*:string {
    .fileInfo?:void {
        .lastModified:long
        .size:long
        .absolutePath:string
        .isHidden:bool
        .isDirectory:bool
    }
}
```

Since it seems that if you have any of the fields, you are sure to have all of them.